hbase-server/src/main/java/org/apache/hadoop/hbase/wal/WALPrettyPrinter.java

```diff
@@ -326,15 +340,19 @@ public void processFile(final Configuration conf, final Path p)
           out.print(GSON.toJson(txn));
         } else {
           // Pretty output, complete with indentation by atomic action
-          out.println(String.format(outputTmpl,
+          if (!outputOnlyRowKey) {
+            out.println(String.format(outputTmpl,
               txn.get("sequence"), txn.get("table"), txn.get("region"), new Date(writeTime)));
+          }
           for (int i = 0; i < actions.size(); i++) {
             Map<String, Object> op = actions.get(i);
-            printCell(out, op, outputValues);
+            printCell(out, op, outputOnlyRowKey, outputValues);
           }
         }
-        out.println("edit heap size: " + entry.getEdit().heapSize());
-        out.println("position: " + log.getPosition());
+        if (!outputOnlyRowKey) {
```

Comment:
Why we are removing this info from the output? Is it an additional change not mentioned on the jira original description?